### PR TITLE
Feature/795 fe editorial detail top

### PR DESCRIPTION
**Pull request by nicklee**
The top area of the new editorial detail design as per these designs:
https://www.figma.com/file/XWY63bn8fmek4LbAtcxh2p/RCA-Phase-3-shared?node-id=5261%3A22577

Ticket URL:
https://projects.torchbox.com/projects/rca-website-rebuild/tickets/795

Example pages:
http://localhost:8000/pattern-library/render-pattern/patterns/pages/editorial/editorial_detail.html
https://rca-development.herokuapp.com/pattern-library/render-pattern/patterns/pages/editorial/editorial_detail.html

http://localhost:8000/pattern-library/render-pattern/patterns/pages/editorial/editorial_detail--no-video.html
https://rca-development.herokuapp.com/pattern-library/render-pattern/patterns/pages/editorial/editorial_detail--no-video.html

Screenshot 2021-06-29 at 12 27 22

Screenshot 2021-06-29 at 12 27 50

### CODE REVIEW

File: rca/project_styleguide/templates/patterns/molecules/key-details/key-details--editorial.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                {{ page.read_time }}
+            </li>
+        </ul>
+    </div>
+
+    {% if page.press_pack %}
+        <div class="key-details__section key-details__section--press-pack">
+            <h3 class="body body--two key-details__sub-heading">Press pack</h3>
+            <ul class="key-details__list">
+                    <li class="key-details__list-item">
```
Comment (chris-lawton) [inline rca/project_styleguide/templates/patterns/molecules/key-details/key-details--editorial.html:57]: nitpick: indent

**Comment by chris-lawton** [comment]
Looks fine to me - nice and tidy code as per usual : )

One thing I did spot when testing on dev is when tabbing through the page it seems like you can tab to the elements in the hidden video modal. Probably one for a new ticket as i'm sure it happens elsewhere on the site. The fix could be something as simple as adding `visibility: hidden` to the video elements when they're hidden.

**Comment by nicklee** [comment]
Thanks dude, I've added a ticket for the modal issue: https://projects.torchbox.com/projects/rca-website-rebuild/tickets/863

File: rca/project_styleguide/templates/patterns/molecules/key-details/key-details--editorial.html (hunk 1 above)
Comment (nicklee) [reply rca/project_styleguide/templates/patterns/molecules/key-details/key-details--editorial.html]: Fixed :)

**Pull request merged**